views/webrtc-view/src/components/Transcriber/Transcriber.tsx

```diff
@@ -244,8 +242,9 @@ export default function Transcriber({ source, perspective, webRTC }: Props) {
     mediaStreamSource.connect(workletNode);
     workletNode.port.onmessage = (event) => {
       if (listening.current) {
-        client.ai.feedTranscriptionStream(fastStreamId.current, Array.from(event.data));
-        client.ai.feedTranscriptionStream(streamId.current, Array.from(event.data));
+        const audioData = Array.from(event.data);
+        client.ai.feedTranscriptionStream(fastStreamId.current, audioData);
+        client.ai.feedTranscriptionStream(streamId.current, audioData);
```

Comment:
We could actually use the new `multiple transcription streams` feature here now: https://github.com/coasys/ad4m/pull/590